certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json

```diff
@@ -0,0 +1,800 @@
+        "summary": "Get mail agent",
+        "description": "Get mail agent",
+        "operationId": "get_mail_agent",
+        "x-ms-visibility": "internal",
+        "parameters": []
+      }
+    },
+    "/portal/v1.0/mailagents/{mailagent_key}/bounce-address": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "bounce_address": {
+                        "type": "string",
+                        "description": "Bounce address"
+                      },
+                      "is_default": {
+                        "type": "boolean",
+                        "description": "Is default"
+                      }
+                    }
+                  },
+                  "description": "data"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get bounce address",
+        "description": "Get bounce address",
+        "x-ms-visibility": "internal",
+        "operationId": "get_bounce_address",
+        "parameters": [
+          {
+            "name": "mailagent_key",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "mail agent key",
+            "x-ms-summary": "mail agent key",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/v1.0/email": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
+          },
+          "201": {
+            "description": "OK",
+            "schema": {}
+          }
+        },
+        "summary": "Send mail",
+        "description": "Send mail",
+        "operationId": "send_mail",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "mailagent_key": {
+                  "type": "string",
+                  "description": "Mail Agent",
+                  "x-ms-summary": "Mail Agent",
+                  "x-ms-dynamic-list": {
+                    "operationId": "get_mail_agent",
+                    "itemValuePath": "mailagent_key",
+                    "itemsPath": "data",
+                    "itemTitlePath": "mailagent_name"
+                  }
+                },
+                "bounce_address": {
+                  "type": "string",
+                  "description": "Bounce addresss",
```

Comment:
@zohomail all files should be free of grammatical errors and typos. can you please change "addresss" to "address".